test/fileGatherer.test.ts

```diff
@@ -7,14 +7,17 @@ import FileGatherer from "../src/fileGatherer";
 suite("File Gatherer Tests", () => {
     let fileGatherer: FileGatherer
     let fsStatSync: Sinon.SinonStub;
-    let getExcludeRegEx: Sinon.SinonStub;;
+    let getExcludeRegEx: Sinon.SinonStub;
+    let getExtensionsRegEx: Sinon.SinonStub;
     const directoryObject = { isDirectory() {return true}, isFile() {return false}}
     const fileObject = { isDirectory() {return false}, isFile() {return true}}
 
     suiteSetup(() => {
         fileGatherer = new FileGatherer();
         getExcludeRegEx = Sinon.stub(fileGatherer, "getExcludeRegEx");
         getExcludeRegEx.returns("(\\.spec\\.|\\.test\\.|\\.e2e\\.)");
+        getExtensionsRegEx = Sinon.stub(fileGatherer, "getExtensionsRegEx");
+        getExtensionsRegEx.returns("\\.tsx?$");
```

Comment:
I matched your convention here, but wouldn’t it be better not to stub the default to ensure the value you have in `package.json` is getting picked up correctly?